### ISSUE TITLE
Match code signing work locally but fails on CI runner (profile doesn't include signing certificate)

### ISSUE THREAD

**Issue by JasperStam** [fastlane/fastlane#11149]
### New Issue Checklist

- [x] Updated fastlane to the latest version
- [x] I read the [Contribution Guidelines](https://github.com/fastlane/fastlane/blob/master/CONTRIBUTING.md)
- [x] I read [docs.fastlane.tools](https://docs.fastlane.tools)
- [x] I searched for [existing GitHub issues](https://github.com/fastlane/fastlane/issues)

### Issue Description
Code signing fails on the gitlab ci-runner, installed on a mac mini, but succeeds locally. I find this strange because I configured match, and they both use exactly the same cert & provisioning profile (UUID: b759f109-1928-41d8-acfe-09891b0274db).

CI pipeline build output that shows the exact error:
```
[17:53:57]: ▸ === BUILD TARGET trs OF PROJECT trs WITH CONFIGURATION Release ===
[17:53:57]: ▸ Check dependencies
[17:53:57]: ▸ Code Signing Error: Provisioning profile "nl.codeyellow.fastlane AppStore" doesn't include signing certificate "iPhone Distribution: Code Yellow BV (2U87FAMFLM)".
[17:53:57]: ▸ Code Signing Error: Code signing is required for product type 'Application' in SDK 'iOS 11.2'
```

I should mention that I used [this guide](http://macoscope.com/blog/simplify-your-life-with-fastlane-match/#migration) to migrate existing certificates & provisioning profile to `match`. This however decrypts fine on both environments.

##### Complete output when running fastlane, including the stack trace and command used

<details><summary>Local output</summary>
[japser@exogorth Work/trs-v2 .]% bundle exec fastlane ios beta > local-output
[17:55:23]: -------------------------------------------------
[17:55:23]: --- Step: Verifying required fastlane version ---
[17:55:23]: -------------------------------------------------
[17:55:23]: Your fastlane version 2.68.2 matches the minimum requirement of 2.28.3  ✅
[17:55:23]: ------------------------------
[17:55:23]: --- Step: default_platform ---
[17:55:23]: ------------------------------
[17:55:23]: Driving the lane 'ios beta' 🚀
[17:55:23]: -------------------
[17:55:23]: --- Step: match ---
[17:55:23]: -------------------
[17:55:23]: Successfully loaded '/Users/japser/Work/trs-v2/fastlane/Matchfile' 📄

+---------+----------------------------------------------+
|      Detected Values from './fastlane/Matchfile'       |
+---------+----------------------------------------------+
| git_url | git@gitlab.com:CodeYellowBV/certificates.git |
| type    | development                                  |
+---------+----------------------------------------------+

+-----------------------+----------------------------------------------+
|                       Summary for match 2.68.2                       |
+-----------------------+----------------------------------------------+
| type                  | appstore                                     |
| git_url               | git@gitlab.com:CodeYellowBV/certificates.git |
| git_branch            | master                                       |
| app_identifier        | nl.codeyellow.fastlane                       |
| username              | apple@codeyellow.nl                          |
| keychain_name         | login.keychain                               |
| readonly              | false                                        |
| team_id               | 2U87FAMFLM                                   |
| verbose               | false                                        |
| force                 | false                                        |
| skip_confirmation     | false                                        |
| shallow_clone         | false                                        |
| clone_branch_directly | false                                        |
| force_for_new_devices | false                                        |
| skip_docs             | false                                        |
| platform              | ios                                          |
+-----------------------+----------------------------------------------+

[17:55:23]: Cloning remote git repo...
[17:55:23]: If cloning the repo takes too long, you can use the `clone_branch_directly` option in match.
[17:55:26]: 🔓  Successfully decrypted certificates repo
[17:55:26]: Verifying that the certificate and profile are still valid on the Dev Portal...
[17:55:28]: Installing certificate...

+-------------------+--------------------------------------------------+
|                        Installed Certificate                         |
+-------------------+--------------------------------------------------+
| User ID           | 2U87FAMFLM                                       |
| Common Name       | iPhone Distribution: Code Yellow BV (2U87FAMFLM) |
| Organisation Unit | 2U87FAMFLM                                       |
| Organisation      | Code Yellow BV                                   |
| Country           | US                                               |
| Start Datetime    | Feb 17 14:36:24 2017 GMT                         |
| End Datetime      | Feb 17 14:36:24 2018 GMT                         |
+-------------------+--------------------------------------------------+

[17:55:29]: Installing provisioning profile...

+---------------------+---------------------------------------------+---------------------------------------------+
|                                         Installed Provisioning Profile                                          |
+---------------------+---------------------------------------------+---------------------------------------------+
| Parameter           | Environment Variable                        | Value                                       |
+---------------------+---------------------------------------------+---------------------------------------------+
| App Identifier      |                                             | nl.codeyellow.fastlane                      |
| Type                |                                             | appstore                                    |
| Platform            |                                             | ios                                         |
| Profile UUID        | sigh_nl.codeyellow.fastlane_appstore        | b759f109-1928-41d8-acfe-09891b0274db        |
| Profile Name        | sigh_nl.codeyellow.fastlane_appstore_profi  | nl.codeyellow.fastlane AppStore             |
|                     | le-name                                     |                                             |
| Profile Path        | sigh_nl.codeyellow.fastlane_appstore_profi  | /Users/japser/Library/MobileDevice/Provisi  |
|                     | le-path                                     | oning                                       |
|                     |                                             | Profiles/b759f109-1928-41d8-acfe-09891b027  |
|                     |                                             | 4db.mobileprovision                         |
| Development Team ID | sigh_nl.codeyellow.fastlane_appstore_team-  | 2U87FAMFLM                                  |
|                     | id                                          |                                             |
+---------------------+---------------------------------------------+---------------------------------------------+

[17:55:29]: All required keys, certificates and provisioning profiles are installed 🙌
[17:55:29]: Setting Provisioning Profile type to 'app-store'
[17:55:29]: ------------------------------------
[17:55:29]: --- Step: increment_build_number ---
[17:55:29]: ------------------------------------
Current version of project trs is: 
    3

/Users/japser/Work/trs-v2
[17:55:30]: $ cd /Users/japser/Work/trs-v2/ios && agvtool next-version -all && cd -
[17:55:31]: ▸ Setting version of project trs to:
[17:55:31]: ▸ 4.
[17:55:31]: ▸ Also setting CFBundleVersion key (assuming it exists)
[17:55:31]: ▸ Updating CFBundleVersion in Info.plist(s)...
[17:55:31]: ▸ Updated CFBundleVersion in "trs.xcodeproj/../trs-tvOS/Info.plist" to 4
[17:55:31]: ▸ Updated CFBundleVersion in "trs.xcodeproj/../trs-tvOSTests/Info.plist" to 4
[17:55:31]: ▸ Updated CFBundleVersion in "trs.xcodeproj/../trs/Info.plist" to 4
[17:55:31]: ▸ Updated CFBundleVersion in "trs.xcodeproj/../trsTests/Info.plist" to 4
[17:55:31]: ▸ /Users/japser/Work/trs-v2
[17:55:31]: -----------------
[17:55:31]: --- Step: gym ---
[17:55:31]: -----------------
[17:55:32]: $ xcodebuild -list -project ./ios/trs.xcodeproj
[17:55:33]: $ xcodebuild -showBuildSettings -scheme trs -project ./ios/trs.xcodeproj
[17:55:35]: Detected provisioning profile mapping: {:"nl.codeyellow.fastlane"=>"nl.codeyellow.fastlane AppStore"}

+------------------------------------------------------------+-----------------------------------------------------------+
|                                                 Summary for gym 2.68.2                                                 |
+------------------------------------------------------------+-----------------------------------------------------------+
| scheme                                                     | trs                                                       |
| project                                                    | ./ios/trs.xcodeproj                                       |
| export_method                                              | app-store                                                 |
| export_options.provisioningProfiles.nl.codeyellow.fastlane | nl.codeyellow.fastlane AppStore                           |
| destination                                                | generic/platform=iOS                                      |
| output_name                                                | trs                                                       |
| build_path                                                 | /Users/japser/Library/Developer/Xcode/Archives/2017-12-07 |
| clean                                                      | false                                                     |
| output_directory                                           | .                                                         |
| silent                                                     | false                                                     |
| skip_package_ipa                                           | false                                                     |
| buildlog_path                                              | ~/Library/Logs/gym                                        |
| skip_profile_detection                                     | false                                                     |
| xcode_path                                                 | /Applications/Xcode.app                                   |
+------------------------------------------------------------+-----------------------------------------------------------+

[17:55:35]: $ set -o pipefail && xcodebuild -scheme trs -project ./ios/trs.xcodeproj -destination 'generic/platform=iOS' -archivePath /Users/japser/Library/Developer/Xcode/Archives/2017-12-07/trs\ 2017-12-07\ 17.55.35.xcarchive archive | tee /Users/japser/Library/Logs/gym/trs-trs.log | xcpretty
[17:55:36]: ▸ Building React/yoga [Release]
[17:55:36]: ▸ Check Dependencies
[17:55:36]: ▸ Copying /Users/japser/Work/trs-v2/node_modules/react-native/ReactCommon/yoga/yoga/YGEnums.h
[17:55:36]: ▸ Copying /Users/japser/Work/trs-v2/node_modules/react-native/ReactCommon/yoga/yoga/YGNodeList.h
[17:55:36]: ▸ Copying /Users/japser/Work/trs-v2/node_modules/react-native/ReactCommon/yoga/yoga/YGMacros.h
etc...
</details>

<details><summary>CI runner output</summary>
Running with gitlab-runner 10.2.0 (0a75cdd1)
  on fastlane-macmini (0cb12c5d)
Using Shell executor...
section_start:1512665597:prepare_script
Running on ackbar...
section_end:1512665597:prepare_script
section_start:1512665597:get_sources
Fetching changes...
Removing fastlane/report.xml
HEAD is now at 4b52973 Bump version number
From https://gitlab.com/CodeYellowBV/trs-v2
   4b52973..e2d3979  master     -> origin/master
Checking out e2d3979c as master...
Skipping Git submodules setup
section_end:1512665598:get_sources
section_start:1512665598:restore_cache
section_end:1512665598:restore_cache
section_start:1512665598:download_artifacts
section_end:1512665598:download_artifacts
section_start:1512665598:build_script
$ gem install bundler
Successfully installed bundler-1.16.0
Parsing documentation for bundler-1.16.0
Done installing documentation for bundler after 9 seconds
1 gem installed
$ bundle install
Fetching gem metadata from https://rubygems.org/........
Fetching CFPropertyList 2.3.6
Installing CFPropertyList 2.3.6
Using public_suffix 2.0.5
Using addressable 2.5.2
Using babosa 1.0.2
Using bundler 1.16.0
Using claide 1.0.2
Using colored 1.2
Using colored2 3.1.2
Using highline 1.7.10
Using commander-fastlane 4.4.5
Using declarative 0.0.10
Using declarative-option 0.1.0
Using unf_ext 0.0.7.4
Using unf 0.1.4
Using domain_name 0.5.20170404
Using dotenv 2.2.1
Using excon 0.59.0
Using multipart-post 2.0.0
Using faraday 0.13.1
Using http-cookie 1.0.3
Using faraday-cookie_jar 0.0.6
Using faraday_middleware 0.12.2
Using fastimage 2.1.0
Using gh_inspector 1.0.3
Fetching jwt 2.1.0
Installing jwt 2.1.0
Using little-plugger 1.1.4
Using multi_json 1.12.2
Using logging 2.2.2
Using memoist 0.16.0
Using os 0.9.6
Using signet 0.8.1
Using googleauth 0.6.2
Using httpclient 2.8.3
Using mime-types-data 3.2016.0521
Using mime-types 3.1
Using uber 0.1.0
Using representable 3.0.4
Using retriable 3.1.1
Using google-api-client 0.13.6
Using json 2.1.0
Using mini_magick 4.5.1
Using multi_xml 0.6.0
Using plist 3.3.0
Using rubyzip 1.2.1
Using security 0.1.3
Using slack-notifier 1.5.1
Using terminal-notifier 1.8.0
Using unicode-display_width 1.3.0
Using terminal-table 1.8.0
Using tty-screen 0.6.3
Using word_wrap 1.0.0
Using nanaimo 0.2.3
Using xcodeproj 1.5.3
Using rouge 2.0.7
Using xcpretty 0.2.8
Using xcpretty-travis-formatter 1.0.0
Fetching fastlane 2.68.2
Installing fastlane 2.68.2
Bundle complete! 1 Gemfile dependency, 57 gems now installed.
Use `bundle info [gemname]` to see where a bundled gem is installed.
$ bundle exec fastlane beta
[17:53:40]: -------------------------------------------------
[17:53:40]: --- Step: Verifying required fastlane version ---
[17:53:40]: -------------------------------------------------
[17:53:40]: Your fastlane version 2.68.2 matches the minimum requirement of 2.28.3  ✅
[17:53:41]: ------------------------------
[17:53:41]: --- Step: default_platform ---
[17:53:41]: ------------------------------
[17:53:41]: Driving the lane 'ios beta' 🚀
[17:53:41]: -------------------
[17:53:41]: --- Step: match ---
[17:53:41]: -------------------
[17:53:41]: Successfully loaded '/Users/codeyellow/Downloads/builds/0cb12c5d/0/CodeYellowBV/trs-v2/fastlane/Matchfile' 📄

+---------+----------------------------------------------+
|      Detected Values from './fastlane/Matchfile'       |
+---------+----------------------------------------------+
| git_url | git@gitlab.com:CodeYellowBV/certificates.git |
| type    | development                                  |
+---------+----------------------------------------------+

+-----------------------+----------------------------------------------+
|                       Summary for match 2.68.2                       |
+-----------------------+----------------------------------------------+
| type                  | appstore                                     |
| git_url               | git@gitlab.com:CodeYellowBV/certificates.git |
| git_branch            | master                                       |
| app_identifier        | nl.codeyellow.fastlane                       |
| username              | apple@codeyellow.nl                          |
| keychain_name         | login.keychain                               |
| readonly              | false                                        |
| team_id               | 2U87FAMFLM                                   |
| verbose               | false                                        |
| force                 | false                                        |
| skip_confirmation     | false                                        |
| shallow_clone         | false                                        |
| clone_branch_directly | false                                        |
| force_for_new_devices | false                                        |
| skip_docs             | false                                        |
| platform              | ios                                          |
+-----------------------+----------------------------------------------+

[17:53:41]: Cloning remote git repo...
[17:53:41]: If cloning the repo takes too long, you can use the `clone_branch_directly` option in match.
[17:53:43]: 🔓  Successfully decrypted certificates repo
[17:53:43]: Verifying that the certificate and profile are still valid on the Dev Portal...
[17:53:46]: Installing certificate...

+-------------------+--------------------------------------------------+
|                        Installed Certificate                         |
+-------------------+--------------------------------------------------+
| User ID           | 2U87FAMFLM                                       |
| Common Name       | iPhone Distribution: Code Yellow BV (2U87FAMFLM) |
| Organisation Unit | 2U87FAMFLM                                       |
| Organisation      | Code Yellow BV                                   |
| Country           | US                                               |
| Start Datetime    | Feb 17 14:36:24 2017 GMT                         |
| End Datetime      | Feb 17 14:36:24 2018 GMT                         |
+-------------------+--------------------------------------------------+

[17:53:48]: Installing provisioning profile...

+---------------------+---------------------------------------------------+-------------------------------------------------------------------------------------------------------------------+
|                                                                               Installed Provisioning Profile                                                                                |
+---------------------+---------------------------------------------------+-------------------------------------------------------------------------------------------------------------------+
| Parameter           | Environment Variable                              | Value                                                                                                             |
+---------------------+---------------------------------------------------+-------------------------------------------------------------------------------------------------------------------+
| App Identifier      |                                                   | nl.codeyellow.fastlane                                                                                            |
| Type                |                                                   | appstore                                                                                                          |
| Platform            |                                                   | ios                                                                                                               |
| Profile UUID        | sigh_nl.codeyellow.fastlane_appstore              | b759f109-1928-41d8-acfe-09891b0274db                                                                              |
| Profile Name        | sigh_nl.codeyellow.fastlane_appstore_profile-name | nl.codeyellow.fastlane AppStore                                                                                   |
| Profile Path        | sigh_nl.codeyellow.fastlane_appstore_profile-path | /Users/codeyellow/Library/MobileDevice/Provisioning Profiles/b759f109-1928-41d8-acfe-09891b0274db.mobileprovision |
| Development Team ID | sigh_nl.codeyellow.fastlane_appstore_team-id      | 2U87FAMFLM                                                                                                        |
+---------------------+---------------------------------------------------+-------------------------------------------------------------------------------------------------------------------+

[17:53:49]: All required keys, certificates and provisioning profiles are installed 🙌
[17:53:49]: Setting Provisioning Profile type to 'app-store'
[17:53:49]: ------------------------------------
[17:53:49]: --- Step: increment_build_number ---
[17:53:49]: ------------------------------------
Current version of project trs is: 
    2

/Users/codeyellow/Downloads/builds/0cb12c5d/0/CodeYellowBV/trs-v2
[17:53:49]: $ cd /Users/codeyellow/Downloads/builds/0cb12c5d/0/CodeYellowBV/trs-v2/ios && agvtool next-version -all && cd -
[17:53:50]: ▸ Setting version of project trs to:
[17:53:50]: ▸ 3.
[17:53:50]: ▸ Also setting CFBundleVersion key (assuming it exists)
[17:53:50]: ▸ Updating CFBundleVersion in Info.plist(s)...
[17:53:50]: ▸ Updated CFBundleVersion in "trs.xcodeproj/../trs-tvOS/Info.plist" to 3
[17:53:50]: ▸ Updated CFBundleVersion in "trs.xcodeproj/../trs-tvOSTests/Info.plist" to 3
[17:53:50]: ▸ Updated CFBundleVersion in "trs.xcodeproj/../trs/Info.plist" to 3
[17:53:50]: ▸ Updated CFBundleVersion in "trs.xcodeproj/../trsTests/Info.plist" to 3
[17:53:50]: ▸ /Users/codeyellow/Downloads/builds/0cb12c5d/0/CodeYellowBV/trs-v2
[17:53:50]: -----------------
[17:53:50]: --- Step: gym ---
[17:53:50]: -----------------
[17:53:50]: $ xcodebuild -list -project ./ios/trs.xcodeproj
[17:53:52]: $ xcodebuild -showBuildSettings -scheme trs -project ./ios/trs.xcodeproj
[17:53:55]: Detected provisioning profile mapping: {:"nl.codeyellow.fastlane"=>"nl.codeyellow.fastlane AppStore"}

+------------------------------------------------------------+---------------------------------------------------------------+
|                                                   Summary for gym 2.68.2                                                   |
+------------------------------------------------------------+---------------------------------------------------------------+
| scheme                                                     | trs                                                           |
| project                                                    | ./ios/trs.xcodeproj                                           |
| export_method                                              | app-store                                                     |
| export_options.provisioningProfiles.nl.codeyellow.fastlane | nl.codeyellow.fastlane AppStore                               |
| destination                                                | generic/platform=iOS                                          |
| output_name                                                | trs                                                           |
| build_path                                                 | /Users/codeyellow/Library/Developer/Xcode/Archives/2017-12-07 |
| clean                                                      | false                                                         |
| output_directory                                           | .                                                             |
| silent                                                     | false                                                         |
| skip_package_ipa                                           | false                                                         |
| buildlog_path                                              | ~/Library/Logs/gym                                            |
| skip_profile_detection                                     | false                                                         |
| xcode_path                                                 | /Applications/Xcode.app                                       |
+------------------------------------------------------------+---------------------------------------------------------------+

[17:53:55]: $ set -o pipefail && xcodebuild -scheme trs -project ./ios/trs.xcodeproj -destination 'generic/platform=iOS' -archivePath /Users/codeyellow/Library/Developer/Xcode/Archives/2017-12-07/trs\ 2017-12-07\ 17.53.55.xcarchive archive | tee /Users/codeyellow/Library/Logs/gym/trs-trs.log | xcpretty
[17:53:56]: ▸ Building trs/trs [Release]
[17:53:56]: ▸ Check Dependencies
[17:53:57]: ▸ ** ARCHIVE FAILED **
[17:53:57]: ▸ The following build commands failed:
[17:53:57]: ▸ Check dependencies
[17:53:57]: ▸ (1 failure)
▸ Building trs/trs [Release]
▸ Check Dependencies
** ARCHIVE FAILED **

The following build commands failed:
Check dependencies
(1 failure)
[17:53:57]: Exit status: 65

+---------------+-------------------------+
|            Build environment            |
+---------------+-------------------------+
| xcode_path    | /Applications/Xcode.app |
| gym_version   | 2.68.2                  |
| export_method | app-store               |
| sdk           | iPhoneOS11.2.sdk        |
+---------------+-------------------------+

[17:53:57]: ▸ === BUILD TARGET trs OF PROJECT trs WITH CONFIGURATION Release ===
[17:53:57]: ▸ Check dependencies
[17:53:57]: ▸ Code Signing Error: Provisioning profile "nl.codeyellow.fastlane AppStore" doesn't include signing certificate "iPhone Distribution: Code Yellow BV (2U87FAMFLM)".
[17:53:57]: ▸ Code Signing Error: Code signing is required for product type 'Application' in SDK 'iOS 11.2'
[17:53:57]: 
[17:53:57]: ⬆️  Check out the few lines of raw `xcodebuild` output above for potential hints on how to solve this error
[17:53:57]: 📋  For the complete and more detailed error log, check the full log at:
[17:53:57]: 📋  /Users/codeyellow/Library/Logs/gym/trs-trs.log
[17:53:57]: 
[17:53:57]: Looks like fastlane ran into a build/archive error with your project
[17:53:57]: It's hard to tell what's causing the error, so we wrote some guides on how
[17:53:57]: to troubleshoot build and signing issues: https://docs.fastlane.tools/codesigning/getting-started/
[17:53:57]: Before submitting an issue on GitHub, please follow the guide above and make
[17:53:57]: sure your project is set up correctly.
[17:53:57]: fastlane uses `xcodebuild` commands to generate your binary, you can see the
[17:53:57]: the full commands printed out in yellow in the above log.
[17:53:57]: Make sure to inspect the output above, as usually you'll find more error information there
[17:53:57]: 
+------------------------------------+---------------------------------------------------------------+
|                                            Lane Context                                            |
+------------------------------------+---------------------------------------------------------------+
| DEFAULT_PLATFORM                   | ios                                                           |
| PLATFORM_NAME                      | ios                                                           |
| LANE_NAME                          | ios beta                                                      |
| SIGH_PROFILE_TYPE                  | app-store                                                     |
| MATCH_PROVISIONING_PROFILE_MAPPING | {"nl.codeyellow.fastlane"=>"nl.codeyellow.fastlane AppStore"} |
| BUILD_NUMBER                       | 3                                                             |
+------------------------------------+---------------------------------------------------------------+
[17:53:57]: Error building the application - see the log above

+------+-------------------------------------+-------------+
|                     fastlane summary                     |
+------+-------------------------------------+-------------+
| Step | Action                              | Time (in s) |
+------+-------------------------------------+-------------+
| 1    | Verifying required fastlane version | 0           |
| 2    | default_platform                    | 0           |
| 3    | match                               | 7           |
| 4    | increment_build_number              | 1           |
| 💥   | gym                                 | 6           |
+------+-------------------------------------+-------------+

[17:53:57]: fastlane finished with errors

[!] Error building the application - see the log above
section_end:1512665638:build_script
section_start:1512665638:after_script
section_end:1512665638:after_script
ERROR: Job failed: exit status 1

</details>

### Environment

<details><summary>✅ fastlane environment (local) ✅</summary>

### Stack

| Key                         | Value                                       |
| --------------------------- | ------------------------------------------- |
| OS                          | 10.12.6                                     |
| Ruby                        | 2.4.2                                       |
| Bundler?                    | true                                        |
| Git                         | git version 2.13.6 (Apple Git-96)           |
| Installation Source         | /usr/local/bin/fastlane                     |
| Host                        | Mac OS X 10.12.6 (16G29)                    |
| Ruby Lib Dir                | /usr/local/Cellar/ruby/2.4.2_1/lib          |
| OpenSSL Version             | OpenSSL 1.0.2l  25 May 2017                 |
| Is contained                | false                                       |
| Is homebrew                 | false                                       |
| Is installed via Fabric.app | false                                       |
| Xcode Path                  | /Applications/Xcode.app/Contents/Developer/ |
| Xcode Version               | 9.1                                         |

### System Locale

| Variable | Value       |   |
| -------- | ----------- | - |
| LANG     | nl_NL.UTF-8 | ✅ |
| LC_ALL   | nl_NL.UTF-8 | ✅ |
| LANGUAGE |             |   |

### fastlane files:

<details><summary>`./fastlane/Fastfile`</summary>

```ruby
# Customise this file, documentation can be found here:
# https://github.com/fastlane/fastlane/tree/master/fastlane/docs
# All available actions: https://docs.fastlane.tools/actions
# can also be listed using the `fastlane actions` command

# Change the syntax highlighting to Ruby
# All lines starting with a # are ignored when running `fastlane`

# If you want to automatically update fastlane if a new version is available:
# update_fastlane

# This is the minimum version number required.
# Update this, if you use features of a newer version
fastlane_version "2.28.3"

default_platform :ios

platform :ios do
  before_all do
    # ENV["SLACK_URL"] = "https://hooks.slack.com/services/..."
    
    
  end

  desc "Runs all the tests"
  lane :test do
    scan
  end

  desc "Submit a new Beta Build to Apple TestFlight"
  desc "This will also make sure the profile is up to date"
  lane :beta do
    match(type: "appstore") # more information: https://codesigning.guide
    increment_build_number(xcodeproj: "./ios/trs.xcodeproj")
    gym(scheme: "trs", project: "./ios/trs.xcodeproj")
    pilot

    # sh "your_script.sh"
    # You can also use other beta testing services here (run `fastlane actions`)
  end

  desc "Deploy a new version to the App Store"
  lane :release do
    # match(type: "appstore")
    # snapshot
    gym # Build your app - more options available
    deliver(force: true)
    # frameit
  end

  # You can define as many lanes as you want

  after_all do |lane|
    # This block is called, only if the executed lane was successful

    # slack(
    #   message: "Successfully deployed new App Update."
    # )
  end

  error do |lane, exception|
    # slack(
    #   message: exception.message,
    #   success: false
    # )
  end
end


# More information about multiple platforms in fastlane: https://github.com/fastlane/fastlane/blob/master/fastlane/docs/Platforms.md
# All available actions: https://docs.fastlane.tools/actions

# fastlane reports which actions are used
# No personal data is recorded. Learn more at https://github.com/fastlane/enhancer

```
</details>

<details><summary>`./fastlane/Appfile`</summary>

```ruby
app_identifier "nl.codeyellow.fastlane" # The bundle identifier of your app
apple_id "apple@codeyellow.nl" # Your Apple email address

team_id "2U87FAMFLM" # Developer Portal Team ID

# you can even provide different app identifiers, Apple IDs and team names per lane:
# More information: https://github.com/fastlane/fastlane/blob/master/fastlane/docs/Appfile.md

```
</details>

### fastlane gems

| Gem      | Version | Update-Status |
| -------- | ------- | ------------- |
| fastlane | 2.68.2  | ✅ Up-To-Date  |

### Loaded fastlane plugins:

**No plugins Loaded**

<details><summary><b>Loaded gems</b></summary>

| Gem                       | Version      |
| ------------------------- | ------------ |
| did_you_mean              | 1.1.0        |
| bundler                   | 1.16.0       |
| io-console                | 0.4.6        |
| CFPropertyList            | 2.3.6        |
| public_suffix             | 2.0.5        |
| addressable               | 2.5.2        |
| babosa                    | 1.0.2        |
| claide                    | 1.0.2        |
| colored                   | 1.2          |
| colored2                  | 3.1.2        |
| highline                  | 1.7.10       |
| commander-fastlane        | 4.4.5        |
| declarative               | 0.0.10       |
| declarative-option        | 0.1.0        |
| unf_ext                   | 0.0.7.4      |
| unf                       | 0.1.4        |
| domain_name               | 0.5.20170404 |
| dotenv                    | 2.2.1        |
| excon                     | 0.59.0       |
| multipart-post            | 2.0.0        |
| faraday                   | 0.13.1       |
| http-cookie               | 1.0.3        |
| faraday-cookie_jar        | 0.0.6        |
| faraday_middleware        | 0.12.2       |
| fastimage                 | 2.1.0        |
| gh_inspector              | 1.0.3        |
| jwt                       | 2.1.0        |
| little-plugger            | 1.1.4        |
| multi_json                | 1.12.2       |
| logging                   | 2.2.2        |
| memoist                   | 0.16.0       |
| os                        | 0.9.6        |
| signet                    | 0.8.1        |
| googleauth                | 0.6.2        |
| httpclient                | 2.8.3        |
| mime-types-data           | 3.2016.0521  |
| mime-types                | 3.1          |
| uber                      | 0.1.0        |
| representable             | 3.0.4        |
| retriable                 | 3.1.1        |
| google-api-client         | 0.13.6       |
| json                      | 2.1.0        |
| mini_magick               | 4.5.1        |
| multi_xml                 | 0.6.0        |
| plist                     | 3.3.0        |
| rubyzip                   | 1.2.1        |
| security                  | 0.1.3        |
| slack-notifier            | 1.5.1        |
| terminal-notifier         | 1.8.0        |
| unicode-display_width     | 1.3.0        |
| terminal-table            | 1.8.0        |
| tty-screen                | 0.6.3        |
| word_wrap                 | 1.0.0        |
| nanaimo                   | 0.2.3        |
| xcodeproj                 | 1.5.3        |
| rouge                     | 2.0.7        |
| xcpretty                  | 0.2.8        |
| xcpretty-travis-formatter | 1.0.0        |
</details>

*generated on:* **2017-12-07**
</details>

<details><summary>✅ fastlane environment (on ci runner)✅</summary>

### Stack

| Key                         | Value                                       |
| --------------------------- | ------------------------------------------- |
| OS                          | 10.13.1                                     |
| Ruby                        | 2.4.2                                       |
| Bundler?                    | false                                       |
| Git                         | git version 2.15.1                          |
| Installation Source         | /usr/local/bin/fastlane                     |
| Host                        | Mac OS X 10.13.1 (17B1003)                  |
| Ruby Lib Dir                | /usr/local/Cellar/ruby/2.4.2_1/lib          |
| OpenSSL Version             | OpenSSL 1.0.2l  25 May 2017                 |
| Is contained                | false                                       |
| Is homebrew                 | false                                       |
| Is installed via Fabric.app | false                                       |
| Xcode Path                  | /Applications/Xcode.app/Contents/Developer/ |
| Xcode Version               | 9.2                                         |

### System Locale

| Variable | Value       |   |
| -------- | ----------- | - |
| LANG     | en_US.UTF-8 | ✅ |
| LC_ALL   | en_US.UTF-8 | ✅ |
| LANGUAGE |             |   |

### fastlane files:

**No Fastfile found**

**No Appfile found**

### fastlane gems

| Gem      | Version | Update-Status |
| -------- | ------- | ------------- |
| fastlane | 2.68.2  | ✅ Up-To-Date  |

### Loaded fastlane plugins:

**No plugins Loaded**

<details><summary><b>Loaded gems</b></summary>

| Gem                       | Version      |
| ------------------------- | ------------ |
| did_you_mean              | 1.1.0        |
| slack-notifier            | 1.5.1        |
| claide                    | 1.0.2        |
| colored2                  | 3.1.2        |
| nanaimo                   | 0.2.3        |
| xcodeproj                 | 1.5.3        |
| rouge                     | 2.0.7        |
| xcpretty                  | 0.2.8        |
| terminal-notifier         | 1.8.0        |
| unicode-display_width     | 1.3.0        |
| terminal-table            | 1.8.0        |
| plist                     | 3.3.0        |
| public_suffix             | 2.0.5        |
| addressable               | 2.5.2        |
| multipart-post            | 2.0.0        |
| word_wrap                 | 1.0.0        |
| tty-screen                | 0.6.3        |
| babosa                    | 1.0.2        |
| colored                   | 1.2          |
| highline                  | 1.7.10       |
| commander-fastlane        | 4.4.5        |
| excon                     | 0.59.0       |
| faraday                   | 0.13.1       |
| unf_ext                   | 0.0.7.4      |
| unf                       | 0.1.4        |
| domain_name               | 0.5.20170404 |
| http-cookie               | 1.0.3        |
| faraday-cookie_jar        | 0.0.6        |
| fastimage                 | 2.1.0        |
| gh_inspector              | 1.0.3        |
| mini_magick               | 4.5.1        |
| multi_json                | 1.12.2       |
| multi_xml                 | 0.6.0        |
| rubyzip                   | 1.2.1        |
| security                  | 0.1.3        |
| xcpretty-travis-formatter | 1.0.0        |
| dotenv                    | 2.2.1        |
| bundler                   | 1.16.0       |
| faraday_middleware        | 0.12.2       |
| uber                      | 0.1.0        |
| declarative               | 0.0.10       |
| declarative-option        | 0.1.0        |
| representable             | 3.0.4        |
| retriable                 | 3.1.1        |
| mime-types-data           | 3.2016.0521  |
| mime-types                | 3.1          |
| little-plugger            | 1.1.4        |
| logging                   | 2.2.2        |
| memoist                   | 0.16.0       |
| os                        | 0.9.6        |
| signet                    | 0.8.1        |
| googleauth                | 0.6.2        |
| httpclient                | 2.8.3        |
| google-api-client         | 0.13.6       |
| json                      | 2.1.0        |
| io-console                | 0.4.6        |
| openssl                   | 2.0.5        |
</details>

*generated on:* **2017-12-07**
</details>

**Comment by JasperStam** [comment]
Yes I have, I used match and selected the provisioning profiles in xcode

**Comment by JasperStam** [comment]
I managed to figure out what the problem was:
The CI machine had 2 certificates in the keychain with the same name but with a different expiry time. (Probably from when I was playing around with cert & sigh). Even though I manually selected the certificate with the right expiry time, it still picked the wrong one on the CI machine.

To fix this I deleted the duplicate cert from the apple developer portal and deleted the duplicate from the CI machine keychain.

I was debugging the provisioning profiles, but the problem was with the certificates.
